From 7e03d376d96e3a4ce7cc608c54b430dd4fe23bc4 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Mon, 5 Dec 2011 13:53:55 +0800
Subject: [PATCH 0683/2463] ENGR00169388-1 IMX/UART : add DMA fields for
 platform data

add two fields for DMA RX and DMA TX.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/plat-mxc/include/mach/imx-uart.h |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/imx-uart.h b/arch/arm/plat-mxc/include/mach/imx-uart.h
index 45a13ab..21dcdaf 100644
--- a/arch/arm/plat-mxc/include/mach/imx-uart.h
+++ b/arch/arm/plat-mxc/include/mach/imx-uart.h
@@ -32,6 +32,8 @@ struct imxuart_platform_data {
 	unsigned int irda_inv_rx:1;
 	unsigned int irda_inv_tx:1;
 	unsigned short transceiver_delay;
+	unsigned int dma_req_rx;
+	unsigned int dma_req_tx;
 };
 
 #endif
-- 
1.7.7.4

